webknossos/examples/download_tiff_stack.py

```diff
@@ -0,0 +1,30 @@
+from tifffile import imwrite
+
+import webknossos as wk
+
+DATASET_NAME = "l4_sample"
+LAYER_NAME = "color"
+MAG = wk.Mag("4-4-1")
+
+
+def main() -> None:
+    dataset = wk.Dataset.open_remote(DATASET_NAME, organization_id="scalable_minds")
+    mag_view = dataset.get_layer(LAYER_NAME).get_mag(MAG)
+
+    z = mag_view.layer.bounding_box.topleft.z
+    with mag_view.get_buffered_slice_reader() as reader:
+        for slice_data in reader:
```

Comment:
I'd rather have a progress bar instead of the print statement. E.g. using `from rich.progress import track`:

```suggestion
        for slice_data in track(reader):
```

Reply:
I'd rather like to keep it simple. These examples are meant for Python novices to adapt. If we add `track`, I think that is contra-productive. It would be different if it were part of the library.